### PR TITLE
Increment/decrement items

### PR DESCRIPTION
**Pull request by GregJMitchell**
This Branch Does:
- Adds + button to cart page
- Adds - button to cart page
- Allows user to increase or decrease quantity of an item in the cart
All Tests Passing

### CODE REVIEW

File: app/views/cart/show.html.erb [hunk 1]
```diff
@@ -31,6 +31,7 @@
       <td>
         <center>
           <p><%=quantity%></p>
+          <p><%= button_to "+", "/cart/#{item.id}",params: {'operation' => "add"}, method: :patch %></p>
```
Comment (abcdefghijohn) [inline app/views/cart/show.html.erb:34]: Greg showed me the hidden params and used it here. Pretty awesome!

File: app/controllers/cart_controller.rb [hunk 2]
```diff
@@ -20,5 +20,30 @@ def remove_item
...
+    elsif params[:operation] == "subtract"
+      if session[:cart][item.id.to_s] > 1
+        cart.subtract_item(item.id.to_s)
+        flash[:success] = "A #{item.name} was successfully removed from your cart"
+        redirect_to "/cart"
+      else
+        session[:cart].delete(params[:item_id])
+        flash[:success] = "#{item.name} was successfully removed from your cart"
+        redirect_to '/cart'
+      end
```
Comment (jakeheft) [inline app/controllers/cart_controller.rb:44]: Nice conditional/logic on this and accounting for sad path.

File: app/views/cart/show.html.erb (hunk 1 above)
Comment (jakeheft) [reply app/views/cart/show.html.erb:34]: Nice work! I'll have to get schooled on that as well

**Review by jakeheft — APPROVED** [review]
Nice work getting the cart quantity adjustment set up

**Pull request merged**